gun-violence-dashboard/src/pages/About.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
```

Comment:
I think the max width for the text should be more narrow; on wider screens it stretches out a lot is harder to read.

Reply:
I can do that!